Yoast/javascript
# Port keyphrase researches to new morphology research

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Use forms collected from the text as the morphology research.

## Relevant technical choices:

* Specs have been adapted to reflect the new method of retrieving word forms.
* For the moment, the old morphology research is still accessible in the researcher as `morphologyLegacy`. This will be removed in the clean-up phase.

## Test instructions

This PR can be tested by following these steps:

* Run yarn test and make sure that all tests pass.
* Pick your favorite keyphrase-based research. Console.log the topic forms used. Make sure they are forms collected from the paper.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-230
